### DISCUSSION TITLE
震惊！励志成为传奇绳匠的年轻人 竟然止于这一步！

### DISCUSSION THREAD

**Discussion by Tang-Li-1** [share121/inter-knot#1443] [其他]
![Screenshot_20240811_181807](https://github.com/user-attachments/assets/daa8934d-3af4-4c49-8fde-0afde597a4ce)
绳网进不去了昨天还可以今天就一直在加载 挂梯子也进不去只能在github发帖子了

**Comment by Tang-Li-1** [comment]
![Screenshot_20240811_182137](https://github.com/user-attachments/assets/ac97a381-b565-4b9c-8544-82801eb46dda)
变成这样力！

**Comment by FFT66** [comment]
重刷一遍试试

**Comment by Tang-Li-1** [comment]
今天突然进不去了 绳网助手也更新了 梯子也挂了 就是一直加载

**Comment by Mayf1ower** [comment]
梯子换个节点再刷新一下试试  刷新大法解决一切！🤓

**Comment by fairy-B** [comment]
一楼的情况重启一下软件就行了（别忘了登录）
